package.json

```diff
@@ -192,6 +216,88 @@
           "meta.embedded.block.rego": "rego"
         }
       }
+    ],
+    "breakpoints": [
+      {
+        "language": "rego"
+      }
+    ],
+    "debuggers": [
+      {
+        "type": "opa-debug",
+        "label": "OPA Debug",
+        "languages": [
+          "rego"
+        ],
+        "configurationAttributes": {
+          "launch": {
+            "properties": {
+              "command": {
+                "type": "string",
+                "description": "The OPA command to run. E.g. 'eval', 'test'.",
+                "default": "run"
+              },
+              "query": {
+                "type": "string",
+                "default": "data"
+              },
+              "dataPaths": {
+                "type": "array",
+                "items": {
+                  "type": "string"
+                }
+              },
+              "bundlePaths": {
+                "type": "array",
+                "items": {
+                  "type": "string"
+                },
+                "default": ["${workspaceFolder}"]
```

Comment:
Can this depend on another config attribute? If so, `opa.roots` would be a good choice in order to not have users repeating that here.

Reply:
That should be possible to hook up, I think. We can't outright drop it and use `opa.roots` in its stead, as it must be possible to opt out this and only use the `dataPaths` setting. Would it make sense to allow you to specify `"${opa.roots}"`, and defaulting to `["$opa.roots"]`?

Reply:
yeah, as far as I understand the default value for `opa.roots` is already `["${workspaceFolder}"]`, so if you don't configure anything, it'd mean the same thing as above, while allowing you to set this via that option

Reply:
Ah, so basically, if this isn't set, we use `opa.roots` by default. That makes sense 👍 .